bot/seasons/easter/egg_facts.py

```diff
@@ -0,0 +1,53 @@
+import asyncio
+import random
+from json import load
+from pathlib import Path
+
+import discord
+from discord.ext import commands
+
+from bot.constants import Colours
+
+
+class EasterFacts(commands.Cog):
+    """A cog for easter egg facts."""
+    def __init__(self, bot):
+        self.bot = bot
+        self.facts = self.load_json()
+
+    @staticmethod
+    def load_json():
+        """Loading the json data"""
+        p = Path('bot', 'resources', 'easter', 'easter_egg_facts.json')
+        with p.open(encoding="utf8") as f:
+            facts = load(f)
+        return facts
+
+    async def background(self):
+        """A background task that sends a easter egg fact."""
+        channel = self.bot.get_channel(426566445124812815)
+        while True:
+            embed = self.make_embed()
+            await channel.send(embed=embed)
+            await asyncio.sleep(24*60*60)
+
+    @commands.command(name='eggfact', aliases=['fact'])
+    async def easter_facts(self, ctx):
+        """Get easter egg facts."""
+        embed = self.make_embed()
+        await ctx.send(embed=embed)
+
+    def make_embed(self):
+        """Makes a nice embed for the message to be sent."""
+        embed = discord.Embed()
+        embed.colour = Colours.soft_red
+        embed.title = 'Easter Egg Fact'
+        random_fact = random.choice(self.facts)
+        embed.description = random_fact
+        return embed
+
+
+def setup(bot):
+    """Loading the cog"""
```

Comment:
In addition to the newline, other cogs document this in the format """Cog name load.""". A few examples:
```sh
$ ack -A1 '^def setup' bot/
bot/seasons/easter/traditions.py
29:def setup(bot):
30-    """Traditions Cog load."""

bot/seasons/easter/conversationstarters.py
27:def setup(bot):
28-    """Conversation starters Cog load."""
```

In this case, I think a nice docstring would be
```py
"""Egg facts Cog load."""
```
What do you think?

Reply:
I think a bit more description (though may make the string longer) may be more beneficial to the reader who does not know what a cog may be, or what it's supposed to do.

Reply:
> 
> 
> I think a bit more description (though may make the string longer) may be more beneficial to the reader who does not know what a cog may be, or what it's supposed to do.

i think this info is listed in the cog docstring which ull see after i commit the changes.